events/README.md

```diff
@@ -247,6 +248,24 @@ Type(s) of lane used or blocked `vehicle_blocked_lane_types` by the vehicle perf
 
 [Top][toc]
 
+### Payment Type
+
+Strings used to indicate how a curb user paid for a curb event.
+
+| Name              | Description                                            |
+| ----------------- | ------------------------------------------------------ |
+| `cash`            | Bills or coins at a meter. |
+| `credit_card`     | Visa, Mastercard, etc at a meter. |
```

Comment:
Idea to add sub payment type as an option under `payment_type`. Visa, MC, apple pay, etc. Or V2X payment possible.